keyboards/handwired/splittest/bluepill/config.h

```diff
@@ -0,0 +1,18 @@
+// Copyright 2022 dvermd (@dvermd)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "config_common.h"
+
+#define BLUEPILL
+
```

Comment:
I don't see where this is used.
```suggestion
```

Reply:
That's right, it's a leftover of the first version of this PR